eth2/utils/serde_hex/src/lib.rs

```diff
@@ -0,0 +1,80 @@
+// Copied from the file of the same name in the https://github.com/sigp/lighthouse repo with some
+// modifications and deletions.  Specifically, removed all code not needed for this repo.
+
+use hex;
```

Comment:
What modifications did you make exactly? Can you please lay them out/point them out?

Reply:
I'm not sure why you would need to make modifications and copy this over. That's my take.

Reply:
it's used somewhere, let me recall where.  It's possible I can remove this and the thing that depends on it, though, I'll look.

Reply:
Yeah it's necessary for some logic in the `ssz_types` package.  I do think we'll want to replace that package sooner rather than later with a crate, so for now I'm going to keep this in and we can replace it later.